apps/admin-ui/src/clients/authorization/policy/Time.tsx

```diff
@@ -71,15 +71,15 @@ const DateTime = ({ name }: { name: string }) => {
             <SplitItem>
               <DatePicker
                 value={dateTime[1]}
-                onChange={(_, date) => {
+                onChange={(_event, _, date) => {
```

Comment:
Since we're not using `_event` here we can omit the rest of the name of the variable.
```suggestion
                onChange={(_, _, date) => {
```

Reply:
Looks like that causes an error and build failure for "Duplicate identifier"

Reply:
Ah yes of course, let's write out the param names fully then. The linter should not complain as long as a param after is being used.